apps/design/frontend/src/rich_text_editor.tsx

```diff
@@ -147,6 +147,8 @@ function Toolbar({ editor }: { editor: Editor }) {
               })
               .run();
           }}
+          // eslint-disable-next-line no-console
+          onError={(err) => console.error(err)}
```

Comment:
This is existing behavior; since RichTextEditor doesn't specify any minimum dimensions it can only throw on unexpected errors or hitting the image size limit.